CONTRIBUTING.md

```diff
@@ -6,6 +6,21 @@ First of all, many thanks for considering making a contribution to this project.
 
 Please **do not** include any changes to `package-lock.json` in your PRs, even if you are updating `package.json`. These changes are almost impossible to review for security implications. If necessary, `package-lock.json` will be regenerated and committed by a maintainer after your PR is merged.
 
+Please run the test before submitting your pull request.
+
+1. `npm run lint`
+1. `npm test`
+1. `bin/buildAll.sh`
```

Comment:
I would prefer **not** to have this information in the CONTRIBUTING.md apart from the first two - (ignoring current Travis problems), as it should not be necessary for people to run them locally as a first resort, that's what the CI run is designed to catch, and it would have to be kept in sync as more language tests were added.

Reply:
Done